src/screens/SettingsScreen.tsx

```diff
@@ -0,0 +1,174 @@
+import React, {useContext, useState} from 'react'
+import {
+  SafeAreaView,
+  View,
+  StatusBar,
+  StyleSheet,
+  Linking,
+  Platform,
+} from 'react-native'
+import {FormattedMessage, useIntl} from 'react-intl'
+import Picker from 'react-native-picker-select'
+import Icon from 'react-native-vector-icons/MaterialIcons'
+import {getLocales} from 'react-native-localize'
+
+import {containerStyles, colors} from '../styles'
+import {BodyText, BodyHeader} from '../components'
+import {UserContext} from '../providers/user.provider'
+
+function SettingsScreen({navigation}: any) {
+  const {user} = useContext(UserContext)
+  const intl = useIntl()
+  const locales = getLocales().map((locale) => {
+    return {
+      label: intl.formatMessage({
+        id: `translation.${locale.languageCode}`,
```

Comment:
I think this might throw some errors. On first glance anyway.

So this is going to return the locale of the phone, so, it could be Japanese, but we only support 4 languages. So you're going to have to check if it's an accepted language code from us.

Reply:
@SamRichards is correct. I'm going to merge this into iteration 3 anyway for now and then @kenjam97 's next task can be to branch afresh off iteration-3 to implement the 4 language options.

@kenjam97 - have a look at this file as you plan for the next steps of changing locale:

src/effects/use-locale-messages.effect.ts

You'll see that the 4 languages are defined in there. You should start by move that array into a constants file somewhere.